arviz/tests/base_tests/test_data.py

```diff
@@ -537,6 +537,32 @@ def test_stack_unstack(self):
             dataset.stack(z=["c1", "c99"]).unstack(dim="z").posterior, dataset.posterior
         )
 
+    def test_to_dict(self, data_random):
```

Comment:
I'd use https://github.com/arviz-devs/arviz/blob/master/arviz/tests/helpers.py#L171 as it is more complete regarding groups and variables, data_random is a subset by design, otherwise, if it had all groups, it would not be of use to test extend for example, below with the warmup test, data_random has to be used